### DISCUSSION TITLE
启动下载后容易导致qb卡死

### DISCUSSION THREAD

**Discussion by yonoko** [wushuo894/ani-rss#5] [Q&A]
操作步骤：
1. 拉取docker镜像 docker pull wushuo894/ani-rss
2. 编辑 docker-compose.yml 挂载本地番剧目录
3. 执行 docker compose up -d 启动容器并登陆
4. 配置qB下载目录、限制同时下载数为4
5. 依次点击 添加 -> Mikan ，搜索 义妹生活 ，添加订阅沸羊羊字幕组
6. 大约4分钟后开始下载，此阶段qb非常卡顿甚至会崩溃退出
7. 下载完成后可见日志依然有大量qB登陆报错，似乎是误报？既然都能下载为什么还会登陆失败？

![日志截图](https://github.com/user-attachments/assets/652bdc16-d0eb-4410-8db1-8dc8a804f3cb)

**Comment by wushuo894** [comment]
卡死期间确实会登录失败，不影响使用。qb确实挺吃配置的。

**Comment by wushuo894** [comment]
你再收紧一下限制吧

**Comment by yonoko** [comment]
> 卡死期间确实会登录失败，不影响使用。qb确实挺吃配置的。

但是普通添加下载任务不会卡，通过ani-rss发起下载就会卡，这里面是不是有优化空间。另外还是影响使用的，毕竟qB不可能只给ani-rss 独占使用。目前这种情况会干扰其他种子下载。

**Comment by wushuo894** [comment]
qb下载是直接调用的qb网页的api，和网页添加是一样的。不存在只有ani-rss才卡。应该是ani-rss同时添加种子时qb同时获取元数据负载较大

**Comment by wushuo894** [comment]
> > 卡死期间确实会登录失败，不影响使用。qb确实挺吃配置的。
> 
> 但是普通添加下载任务不会卡，通过ani-rss发起下载就会卡，这里面是不是有优化空间。另外还是影响使用的，毕竟qB不可能只给ani-rss 独占使用。目前这种情况会干扰其他种子下载。

日常不会有突然下载很多的场景，订阅不会有啥影响。而且ani-rss的同时下载限制是只针对ani-rss添加到qb的任务的。所以你改成限制1-2个没啥影响

**Comment by wushuo894** [comment]
> > 卡死期间确实会登录失败，不影响使用。qb确实挺吃配置的。
> 
> 但是普通添加下载任务不会卡，通过ani-rss发起下载就会卡，这里面是不是有优化空间。另外还是影响使用的，毕竟qB不可能只给ani-rss 独占使用。目前这种情况会干扰其他种子下载。

tr的占用较少但是他的api很不全面，如果后续tr的api完善后会支持tr的

**Comment by yonoko** [comment]
> > > 卡死期间确实会登录失败，不影响使用。qb确实挺吃配置的。
> > 
> > 
> > 但是普通添加下载任务不会卡，通过ani-rss发起下载就会卡，这里面是不是有优化空间。另外还是影响使用的，毕竟qB不可能只给ani-rss 独占使用。目前这种情况会干扰其他种子下载。
> 
> 日常不会有突然下载很多的场景，订阅不会有啥影响。而且ani-rss的同时下载限制是只针对ani-rss添加到qb的任务的。所以你改成限制1-2个没啥影响

既然这样，使用文档里还是补充一下吧，建议下载限制为1-2

**Comment by wushuo894** [comment]
> > > > 卡死期间确实会登录失败，不影响使用。qb确实挺吃配置的。
> > > 
> > > 
> > > 但是普通添加下载任务不会卡，通过ani-rss发起下载就会卡，这里面是不是有优化空间。另外还是影响使用的，毕竟qB不可能只给ani-rss 独占使用。目前这种情况会干扰其他种子下载。
> > 
> > 
> > 日常不会有突然下载很多的场景，订阅不会有啥影响。而且ani-rss的同时下载限制是只针对ani-rss添加到qb的任务的。所以你改成限制1-2个没啥影响
> 
> 既然这样，使用文档里还是补充一下吧，建议下载限制为1-2

可以
